src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -2172,6 +2173,10 @@ public static class UI
          public static readonly string WatermarkText = "Text";
          public static readonly string WatermarkProperties = "Watermark Properties";
          public static readonly string CreateTableSolubilityAlternative = "Create as pH-Solubility table";
+         public static readonly string ContainerCriteria = "Container Criteria";
+         public static readonly string ForAll = "For All";
+         public static readonly string MoleculeExcludeList = "Molecules Excluded";
```

Comment:
Could be also For All Molecules Except

Reply:
same: better with _molecules_

Reply:
What you proposed: `For All Molecules Except ... `

Reply:
done

Reply:
done